pkg/kudoctl/cmd/init_test.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
 )
 
+// to regenerate the golden files run:
+// go test ./pkg/kudoctl/cmd --update=true
```

Comment:
This is documentation that should be in contribution.md or somewhere else... 

bonus points for just adding a `make golden` task to do this as well :)

Reply:
from slack I see there are different opinions and I will tackle this in separate PR to discuss around this. I personally would go for a `make` target but don't want to hide this feature into this PR. As for now I removed the comments